### PR TITLE
add swap() tests for Pool 5

### PR DESCRIPTION
**Pull request by uri-99**
This PR adds the swap tests corresponding to the Pool 5, resolving issue #166 

Before resolving this PR, the original swap tests PR #152 should be resolved.

### CODE REVIEW

**Review by dubzn — APPROVED** [review]

**Review by dpinones — APPROVED** [review]

File: crates/yas_core/src/tests/utils/swap_cases.cairo [hunk 1]
```diff
@@ -367,7 +367,7 @@ mod SwapTestHelper {
                 amount_specified: IntegerTrait::<i256>::new(1000000000000000000, false),
                 sqrt_price_limit: encode_price_sqrt_200_100(),
             },
-            // swap small amounts in/out
+            //// swap small amounts in/out
```
Comment (rcatalan98) [inline crates/yas_core/src/tests/utils/swap_cases.cairo:370]:
```suggestion
            // swap small amounts in/out
```

File: crates/yas_core/src/tests/test_contracts/test_yas_pool.cairo [hunk 2]
```diff
@@ -1581,7 +1581,57 @@ mod YASPoolTests {
...
+
+            #[test]
+            #[available_gas(200000000000)]
+            #[should_panic(expected: ('SPL', 'ENTRYPOINT_FAILED', 'ENTRYPOINT_FAILED'))]
+            fn test_pool_5_panics_0() {
+                let PANIC_CASE = 0;
+                let pool_case = POOL_CASES()[5];
+                let (success_swap_cases, panic_swap_cases) = SWAP_CASES_POOL_5();
+                let expected_cases =
+                    SWAP_EXPECTED_RESULTS_POOL_5(); //get random case, is never executed
```
Comment (rcatalan98) [inline crates/yas_core/src/tests/test_contracts/test_yas_pool.cairo:1613]: What's the meaning of this?
> //get random case, is never executed

**Review by rcatalan98 — CHANGES_REQUESTED** [review]
Just a few minor changes and questions

File: crates/yas_core/src/tests/test_contracts/test_yas_pool.cairo (hunk 2 above)
Comment (uri-99) [reply crates/yas_core/src/tests/test_contracts/test_yas_pool.cairo:1613]: it was a comment that is no longer necessary, it has now been removed

**Pull request merged**